metadata-lock.md

```diff
@@ -37,7 +33,7 @@ summary: 介绍 TiDB 中元数据锁的概念、原理、实现和影响。
 
 ## 使用元数据锁
 
-使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 启用或者关闭元数据锁特性。
+在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从旧版本升级到 v6.5.0 及之后的版本后，会自动打开元数据锁功能。如果需要关闭元数据锁，可以使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 关闭元数据锁特性。
```

Comment:
```suggestion
在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从 v6.5.0 之前的版本升级到 v6.5.0 及之后的版本时，TiDB 会自动开启元数据锁功能。如果需要关闭元数据锁，你可以使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 关闭元数据锁。
```

Reply:
Please remove [the experimental feature in line 38](https://github.com/wjhuang2016/docs-cn/blob/mdl_enable_default/experimental-features.md?plain=1#L38) and modify the default value in [line 68](https://github.com/wjhuang2016/docs-cn/blob/mdl_enable_default/sql-statements/sql-statement-commit.md?plain=1#L68).